wicket-core/src/test/java/org/apache/wicket/markup/html/form/upload/MultiFileUploadFieldTest.java

```diff
@@ -63,13 +64,16 @@ protected void onSubmit()
 				for (int i = 1; i < 2; i++)
 				{
 					FileUpload fileUpload = uploads.get(i);
-					assertEquals(MultiFileUploadFieldTest.class.getSimpleName()+i+".txt", fileUpload.getClientFileName());
+					assertEquals(MultiFileUploadFieldTest.class.getSimpleName() + (i - 1) + ".txt",
```

Comment:
There's a problem with '(i - 1)'. I've reverted it to simply 'i' and test passes with success.  Can you confirm that this change is not necessary?